src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -149,45 +143,7 @@ export class PointerTool extends BaseTool {
         : undefined;
       const positionedGlyph = sceneController.sceneModel.getSelectedPositionedGlyph();
       if (positionedGlyph?.isUndefined) {
-        // Create a new glyph
-        // TODO: dispatch event and let editor.js handle it
-        this.sceneSettings.selectedGlyph = {
-          ...this.sceneSettings.selectedGlyph,
-          isEditing: false,
-        };
-        // Create a new glyph
-        // Or: ask user if they want to create a new glyph
-        const uniString = makeUPlusStringFromCodePoint(
-          positionedGlyph.character?.codePointAt(0)
-        );
-        const charMsg = positionedGlyph.character
-          ? ` for character “${positionedGlyph.character}” (${uniString})`
-          : "";
-        const result = await dialog(
-          `Create a new glyph “${positionedGlyph.glyphName}”?`,
-          `Click “Create” if you want to create a new glyph named “${positionedGlyph.glyphName}”${charMsg}.`,
-          [
-            { title: "Cancel", resultValue: "no", isCancelButton: true },
-            { title: "Create", resultValue: "ok", isDefaultButton: true },
-          ]
-        );
-        if (result === "ok") {
-          const layerName = "<default>";
-          await this.editor.newGlyph(
-            positionedGlyph.glyphName,
-            positionedGlyph.character?.codePointAt(0),
-            VariableGlyph.fromObject({
-              name: positionedGlyph.glyphName,
-              sources: [{ name: layerName, location: {}, layerName: layerName }],
-              layers: { [layerName]: { glyph: positionedGlyph.glyph.instance } },
-            })
-          );
-          this.sceneSettings.selectedGlyph = {
-            ...this.sceneSettings.selectedGlyph,
-            isEditing: true,
-          };
-          this.sceneSettings.selectedSourceIndex = 0;
-        }
+        sceneController._dispatchEvent("requestedNewGlyphCreate", { positionedGlyph });
```

Comment:
Please do not pass the positionedGlyph to the event, but let the handler get it itself.